src/frontend/tsx/Challenges.tsx

```diff
@@ -201,10 +206,34 @@ export function Challenges () {
               type="text"
               placeholder={'~sampel-palnet'}
               value={who}
-              onChange={(e) => setWho(e.target.value)}
+              onChange={(e) => {
+                  setWho(e.target.value);
+                  setNewOpp(e.target.value);
+              }}
               key={badChallengeAttempts}
               disabled={ challengingFriend }/>
           </div>
+          <div
+            className="new-opp-tally-container"
+            style={
+              newOpp === ''
+              ? {display: 'none'}
+              : !ob.isValidPatp(`${newOpp}`)
+              ? {display: 'none'}
+              : {display: 'block'}
+            }
+          >
+            <p className="new-opp-tally">
+                { newOpp === ''
+                  ? ''
+                  : !ob.isValidPatp(`${newOpp}`)
+                  ? ''
+                  : (tallies.get(`${newOpp}`) === undefined
+                  ? '0 - 0'
+                  : tallies.get(`${newOpp}`))
+                }
```

Comment:
Since the checks for the `<div>` visibility above are already hiding / showing the text as necessary, we only need to check for whether we have a tally for the player. Combined with the above two comments, this looks like:
```
{ tallies.has(`${newOpp}`)
    ? '0 - 0'
    : tallies.get(`${newOpp}`))
}
```